Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.10 1 &
 RUN python3.9 -m ensurepip --default-pip --upgrade && \
     pip install --upgrade pip
 
-RUN pip install --pre torch==2.0.0 --extra-index-url https://download.pytorch.org/whl/cu118
+RUN pip install --pre torch==2.0.0
```

Comment:
do we still need to have it here ?

Reply:
it s a huge dependency, having it here avoid rebuilding most of the image for each change of code?